src/load_tests/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+set(LOAD_TESTS_SRC main.cc
+                   check.h check.cc
+                   provision.h provision.cc
+                   executor.h
+                   context.h context.cc
+                   stats.cc stats.h
+                   sslinit.h sslinit.cc)
+
+if (BUILD_LOAD_TESTS)
+
+    add_executable(ota-load-tests ${LOAD_TESTS_SRC})
+
+    include(ExternalProject)
+    ExternalProject_Add(
+            hdr_histogram
+            URL https://github.com/HdrHistogram/HdrHistogram_c/archive/0.9.6.zip
+            INSTALL_COMMAND ""
+    )
+
+    ExternalProject_Get_Property(hdr_histogram source_dir)
+    set(HDRHISTOGRAM_SOURCE_DIR ${source_dir})
+    ExternalProject_Get_Property(hdr_histogram binary_dir)
+    set(HDRHISTOGRAM_BINARY_DIR ${binary_dir})
+
+    set(HDRHISTOGRAM_LIBS
+            ${HDRHISTOGRAM_BINARY_DIR}/src/${CMAKE_STATIC_LIBRARY_PREFIX}hdr_histogram_static${CMAKE_STATIC_LIBRARY_SUFFIX}
+            )
+
+    add_dependencies(ota-load-tests aktualizr hdr_histogram)
```

Comment:
should be `add_dependencies(ota-load-tests aktualizr hdr_histogram_static)` I think.

Also, there is a gcc warning when compiling hdr_histogram (`-Wmissing-prototypes`). Would be better if we could suppress it.

Reply:
I believe it is dependency on a project, and the project is added as 
```
ExternalProject_Add(
            hdr_histogram
```

Reply:
The other option would be to use git submodules to pull it in.  I'm kind-of allergic to fetching stuff from the internet as during the build process, and since the library is built with cmake anyway, it is easy to just reference it directly. See #737